dohproxy/stub.py

```diff
@@ -21,7 +31,9 @@ def parse_args():
     parser.add_argument(
         '--listen-address',
         default='::1',
-        help='The address the stub should listen on. Default: [%(default)s]'
+        help='Comma-separated list of addresses the stub should listen on. '
+             '"all" for all detected interfaces and addresses (netifaces '
+             'required). Default: [%(default)s]'
```

Comment:
nit: We use `nargs=+` for the server side, see https://github.com/facebookexperimental/doh-proxy/blob/ce782965464d1e65f918a75480dd579eb9f2bb1d/dohproxy/utils.py#L277-L283

Let's use the same approach here.

Reply:
Done.